README.md

```diff
@@ -205,7 +205,7 @@ Our proposal is to replace the vague specification sentence above with [text bas
 
 - It makes it clear how `<dialog>`s interact with `CloseWatcher` instances: they both live in the same per-`Document` close watcher stack.
 
-- It drives interoperability in terms of user- and developer-facing `<dialog>` behavior by providing a more concrete specification, e.g. with regards to how the `<dialog>`'s `cancel` event is sequenced versus the `keydown` event for <kbd>Esc</kbd> keyp presses.
+- It drives interoperability in terms of user- and developer-facing `<dialog>` behavior by providing a more concrete specification, e.g. with regards to how the `<dialog>`'s `cancel` event is sequenced versus the `keydown` event for <kbd>Esc</kbd> keypresses.
```

Comment:
This is not correct

Reply:
This was making it consistent with the rest of the document. Should I perhaps change them to be all "key presses"